storage/store_test.go

```diff
@@ -157,13 +157,12 @@ func TestStoreInitAndBootstrap(t *testing.T) {
 
 	// Make sure the store status exists and is correct.
 	expectedStoreStatus := &proto.StoreStatus{
-		StoreID:   1,
-		NodeID:    1,
-		RaftIDs:   []int64{1},
-		UpdatedAt: 0,
-		StartedAt: 0,
-		UsedBytes: 0,
-		MaxBytes:  0,
+		StoreID:    1,
+		NodeID:     1,
+		RangeCount: 1,
+		StartedAt:  0,
+		UpdatedAt:  0,
+		Stats:      proto.MVCCStats{},
```

Comment:
You can rely on go initializing fields that are not explicitly initialized to the "zero" value. The above 3 lines (`StartedAt`, `UpdateAt` and `Stats`) can be removed.